src/components/button/Button.new.driver.ts

```diff
@@ -0,0 +1,33 @@
+import {ButtonProps} from './ButtonTypes';
```

Comment:
The props should be exported from the component IMO

Reply:
Ethan commented on my PR tat it should be from the types, not sure why it's named `ButtonTypes` and not `types` but it's not related to this PR